x-pack/plugins/security_solution/common/detection_engine/rule_management/api/rules/coverage_overview/response_schema.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+import { NonEmptyArray } from '@kbn/securitysolution-io-ts-types';
+import { CoverageOverviewRuleActivity } from './request_schema';
+
+export type CoverageOverviewRuleData = t.TypeOf<typeof CoverageOverviewRuleData>;
+export const CoverageOverviewRuleData = t.type({
+  name: t.string,
+  activity: CoverageOverviewRuleActivity,
+});
```

Comment:
I think it's missing a field for rule id.

Reply:
Rule data object keyed by its ids. Duplicating an `id` here will just inflate the response. For convenience UI is able to add an `id` per rule after the processing step.